makefiles/simulators/Makefile.aldec

```diff
@@ -94,6 +104,9 @@ $(SIM_BUILD)/runsim.tcl : $(VERILOG_SOURCES) $(VHDL_SOURCES) $(SIM_BUILD)
 	@echo "	puts [read [open sim.log r]]" >> $@
 	@echo "	quit -code 1" >> $@
 	@echo "}" >> $@
+	@echo "@if [string length [array get env LICENSE_QUEUE]] {" >> $@
+	@echo " set LICENSE_QUEUE $$::env(LICENSE_QUEUE)" >> $@
+	@echo "}" >> $@
```

Comment:
Could you add documentation for this feature somewhere, or at least file an issue for it so someone else can document it and it doesn't get lost?